README.md

```diff
@@ -0,0 +1,24 @@
+# Botfuel Webchat Client
+
+Botfuel webchat client designed to work with bots made with Botfuel Bot SDK.
+For a more detailed documentation, see https://app.botfuel.io/docs.
+
+## Install
+
+<script> tag using CDNs
+
+jsDelivr is a global CDN delivery for JavaScript libraries.
+
+To include the latest release:
+
+<script src="https://cdn.jsdelivr.net/npm/botfuel-webchat-client"></script>
+
```

Comment:
Can you already document for older releases ?

Reply:
done 👍